src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,114 @@
+import { randomBytes } from 'crypto';
+import models from '../db/models';
+import sendgrid from '../middlewares/helper';
+
+const { User } = models;
+
+/**
+  * @description class representing reset user password
+  *
+  * @class SendResettoken
+  */
+class SendResetToken {
+  /**
+    * @description - This method is responsible for creating creating password reset token
+    *
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    *
+    * @returns {object} - object representing response message
+    *
+    */
+  static async resetToken(request, response) {
+    try {
+      const { email } = request.body;
+      if (email.trim().length < 1) {
+        return response.status(400).json({
+          status: 'Fail',
+          message: 'The email field cannot be empty'
+        });
+      }
+      const userData = await User.findOne({
+        where: { email }
+      });
+
+      if (!userData) {
+        return response.status(400).json({
+          status: 'Fail',
+          message: 'An email has been sent to your mail',
+        });
+      }
+      const userToken = randomBytes(40).toString('hex');
+      const tokenExpiresIn = Date.now() + 3600000;
```

Comment:
Good Job in implementing the user reset password, I like how you're trying to make use of your own hashing to generate a token. however, did you consider using JWT also considering that's the strategy already used for generating user signup? @okoroemeka

Reply:
Ok. It didn't occur to me to make use of JWT. But I will incorporate it now since that's what the team is already using.